PythonClient.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+
+import sys
+import time, readline, thread
+import datetime
+sys.path.append('./gen-py')
+
+from tutorial import RacconChat
+from tutorial.ttypes import *
+from tutorial.constants import *
+
+from thrift import Thrift
+from thrift.transport import TSocket
+from thrift.transport import TTransport
+from thrift.protocol import TBinaryProtocol
+
+def printMessages(messageArray):
+    for msg in messageArray:
+        shift = 10 - len(msg.user_name)
+        print '[' + msg.time + ']' + (' ' * shift) + msg.user_name + ' : ' + msg.text
+
+def printUsers(usersArray):
+    print 'Users online: ' + ', '.join(map(str, usersArray))
+
+def printCommands():
+    for val in MAP_COMMAND.values():
+        print val
+
+def getCurrentTime(now):
+    current_time = str(now.hour) + ':'
+    if len(str(now.minute)) == 1: current_time += '0'
+    current_time += str(now.minute)
+    return current_time
+
+# thread that get all new messages
+def getMessages(client, user):
+    while True:
+        time.sleep(1)
+        sys.stdout.write('\r' + ' ' * (len(readline.get_line_buffer()) + 2) + '\r')
+        printMessages(client.getNewMessages(user))
+        sys.stdout.write(readline.get_line_buffer())
+        sys.stdout.flush()
+
+def main():
+    transport = TSocket.TSocket('localhost', 9090)
```

Comment:
pass host and port as binary arguments, you can use argparse for this https://docs.python.org/3/library/argparse.html